apps/perf-test-react-components/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@fluentui/perf-test-react-components",
+  "description": "Perf tests for @fluentui/react-components.",
+  "version": "9.0.0-rc.0",
```

Comment:
Set the version to `9.0.0-rc.0` so that v9 dependencies get bumped whenever we have a new v9 release. Similar to this #21805